openshift/openshift-restassured/src/main/java/org/arquillian/cube/openshift/restassured/RestAssuredConfigurator.java

```diff
@@ -0,0 +1,22 @@
+package org.arquillian.cube.openshift.restassured;
+
+import org.jboss.arquillian.config.descriptor.api.ArquillianDescriptor;
+import org.jboss.arquillian.core.api.InstanceProducer;
+import org.jboss.arquillian.core.api.annotation.ApplicationScoped;
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.core.api.annotation.Observes;
+
+public class RestAssuredConfigurator {
+
+    @Inject
+    @ApplicationScoped
+    InstanceProducer<RestAssuredConfiguration> restAssuredConfigurationInstanceProducer;
+
+    // Need to be executed after CubeOpenShiftConfiguration
+    public void configure(@Observes(precedence = -200) ArquillianDescriptor arquillianDescriptor) {
```

Comment:
Why do we need to execute it after CubeOpenshiftConfiguration? Can you add reason for it in documentation?